<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagramPropertiesPage</class>
 <widget class="QWidget" name="DiagramPropertiesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>515</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="ui_groupBoxDiagramProperties">
     <property name="title">
      <string>Diagram Properties</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ui_labelDiagramName">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ui_diagramName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="ui_labelZoom">
            <property name="text">
             <string>Zoom:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ui_zoom">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ui_labelZoomUnit">
            <property name="text">
             <string>%</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="ui_labelWidth">
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ui_width">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>640.000000000000000</double>
            </property>
            <property name="maximum">
             <double>11000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>20.000000000000000</double>
            </property>
            <property name="value">
             <double>1100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="ui_labelHeight">
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="ui_height">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>400.000000000000000</double>
            </property>
            <property name="maximum">
             <double>8000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>20.000000000000000</double>
            </property>
            <property name="value">
             <double>800.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="ui_line">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="ui_checkBoxShowGrid">
          <property name="text">
           <string>Show &amp;grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ui_snapToGrid">
          <property name="text">
           <string>Snap &amp;to grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ui_snapComponentSizeToGrid">
          <property name="whatsThis">
           <string>Snap the size of the components to be a multiple of the grid spacing.
If 'Snap to Grid' is enabled a component will always be aligned with the grid on all 4 sides.</string>
          </property>
          <property name="text">
           <string>Snap component size</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ui_labelGridSpacing">
          <property name="text">
           <string>Grid spacing:  </string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ui_labelX">
          <property name="text">
           <string>X</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ui_gridSpaceX">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ui_labelY">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ui_gridSpaceY">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ui_autoIncrementSequence">
        <property name="text">
         <string>Autoincrement Sequence </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ui_groupBoxDocumentation">
     <property name="title">
      <string>Documentation</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>11</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="KTextEdit" name="ui_documentation"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
